docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -253,7 +253,7 @@ Effective with version 4.7.0, this flag was deprecated and replaced by [`--tunne
 
 <p><small>| OPTIONAL | STRING | <span className="sauceRed">4.6.x</span> <span className="sauceRed">4.7.x</span> <span className="sauceRed">4.8.x</span>| </small></p>
 
-**Description**: Supplies PAC authentication in the format `username:password@host:port`. This option can be used multiple times for each authenticated host in the PAC file.<br/>
+**Description**: Supplies PAC authentication in the format `username:password@host:port`. This option can be used multiple times for each authenticated host in the PAC file. Use a comma separated list when using multiple PAC settings via environment variable.<br/>
```

Comment:
Do we allow spaces? It might be worth mentioning if not, similar to domain formatting https://docs.saucelabs.com/dev/cli/sauce-connect-proxy/#formatting-domains